lib/dentaku/tokenizer.rb

```diff
@@ -4,15 +4,49 @@
 module Dentaku
   class Tokenizer
     NAMES = {
-      operator: { pow: '^', add: '+', subtract: '-', multiply: '*', divide: '/', mod: '%' }.invert,
-      grouping: { open: '(', close: ')', comma: ',' }.invert,
-      dictionary: { open: '{', close: '}', comma: ',' }.invert,
-      list: { open: '[', close: ']', comma: ',' }.invert,
-      case: { open: 'case', close: 'end', then: 'then', when: 'when', else: 'else' }.invert,
+      operator: {
+        '^'  => :pow,
+        '+'  => :add,
+        '-'  => :subtract,
+        '*'  => :multiply,
+        '/'  => :divide,
+        '%'  => :mod,
+        '..' => :range
+      }.freeze,
+      grouping: {
+        '(' => :open,
+        ')' => :close,
+        ',' => :comma,
+      }.freeze,
+      dictionary: {
+        '{' => :open,
+        '}' => :close,
+        ',' => :comma,
+      }.freeze,
+      list: {
+        '[' => :open,
+        ']' => :close,
+        ',' => :comma,
+      }.freeze,
+      case: {
+        'case' => :open,
+        'end' => :close,
+        'then' => :then,
+        'when' => :when,
+        'else' => :else,
+        '..' => :range,
```

Comment:
Is this meant to be under case as well as operator?

Reply:
Nope. Definitely not.